src/api/searchApi.ts

```diff
@@ -420,3 +421,11 @@ export const fetchRegistrationsExport = async (searchParams: URLSearchParams) =>
   const fetchExport = await apiRequest2<string>({ url, headers: { Accept: 'text/csv' } });
   return fetchExport.data;
 };
+
+export const fetchVocabulary = async (customerIdentifier: string) => {
```

Comment:
Vi (prøver) å være konsekvente på naming av sånne ting både i frontend og backend: `id -> URI (https://api.nva.no/publication/<UUID>` og `identifier -> UUID`
```suggestion
export const fetchVocabulary = async (customerId: string) => {
```

Reply:
id/uri er hele adressen, altså `https://api.dev.nva.aws.unit.no/customer/bb3d0c0c-5065-4623-9b98-5810983c2478`, mens identifier/uuid er bare den tilfeldige strengen, altså `bb3d0c0c-5065-4623-9b98-5810983c2478`.
Her er det feks fra responsen når vi henter customer:
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/ae9b5d04-ebe1-4c01-a503-1187b791dac6)

og den strengen som sendes inn her er hele URIen om jeg leser riktig?

Reply:
Ahhh, du har helt rett. Av en eller annen grunn trodde jeg useQuery proddet inn base-url, og at vi kun providet endepunkt 🙈 Burde gått opp et lys da det er user.customerId som sendes inn. 
Jeg endrer 👍

Reply:
du har delvis rett da, for axios (som vi bruker for å gjøre kall) har satt baseUrl til å være rota på apiet, altså `REACT_APP_API_HOST` fra `.env`.
Så du kunne sendt inn bare identfier her også, men da måtte du lagt til base path selv sånn:
```
url: `${CustomerInstitutionApiPath.Customer}/${customerIdentifier}`
```
har ingen sterke meninger om å gjøre det ene eller andre veien, så lenge vi er konsekvente med bruk av `id` og `identifier` 👍